library/src/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ if( WIN32 )
     if( ${CMAKE_BUILD_TYPE} MATCHES "Debug" )
       add_custom_command( TARGET hipsolver
         POST_BUILD
-        COMMAND ${CMAKE_COMMAND} -E copy "${PROJECT_BINARY_DIR}/staging/hipsolver-d.pdb" "${PROJECT_BINARY_DIR}/clients/staging/hipsolver-d.pdb"
+        COMMAND ${CMAKE_COMMAND} -E copy "${PROJECT_BINARY_DIR}/staging/hipsolver.pdb" "${PROJECT_BINARY_DIR}/clients/staging/hipsolver.pdb"
```

Comment:
I am wondering whether this copy can fail if someone sets `CMAKE_DEBUG_POSTFIX` or `CMAKE_EXECUTABLE_SUFFIX` for a build.

Mor precisely, I don't know whether setting `CMAKE_DEBUG_POSTFIX` or `CMAKE_EXECUTABLE_SUFFIX` would change the names of `pdb`s (although I would assume that those should fallow the name assigned to the .libs), and if they do, whether it is in a self-consistent manner. Given that `CMAKE_EXECUTABLE_SUFFIX` also includes the `.exe` in Windows, I would suspect that non-obvious interactions may happen when those two are mixed.

Reply:
It probably will fail. The code that copies files around on Windows is ugly and brittle in many ways.

Just for a bit of background, this operation is performed in the build code because our Windows CI only copies the files from `${PROJECT_BINARY_DIR}/clients/staging/` from the build machine to the test machine. IMO, we should find a better way to handle that. For example, running the tests on the library install tree rather than on the build tree.

Maybe we could add an option so that these copies can be disabled, and ease the transition to an eventual future where they do not exist.

Reply:
I've updated the code to use the `TARGET_PDB_FILE_NAME` generator expression, which should contain the correct name in either case.